runtime/acala/src/lib.rs

```diff
@@ -1018,6 +1018,12 @@ where
 	type Extrinsic = UncheckedExtrinsic;
 }
 
+parameter_type_with_key! {
+	pub MinimumCollateralAmount: |currency_id: CurrencyId| -> Balance {
+		ExistentialDeposits::get(currency_id).saturating_mul(100u128)
```

Comment:
it should be something similar to 

https://github.com/AcalaNetwork/Acala/blob/c4f40d1bfba1405c775ba87f57dd17d309290403/runtime/karura/src/lib.rs#L1917-L1921

because the ExistentialDeposits for KAR / ACA is configured to be infinite

Reply:
I think we want an integration test to confirm

Reply:
Fixed and added an integration test